### PR TITLE
chore(package-json): add files option to package.json

### PR DESCRIPTION
**Pull request by thetimothyp**
In response to this comment: https://github.com/ember-cli/ember-cli/pull/7492#discussion_r157123622

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,12 @@
   "scripts": {
     "test": "mocha tests/**/*-test.js"
   },
+  "files": [
```
Comment (lennyburdette) [inline package.json]: https://docs.npmjs.com/files/package.json#files

> Certain files are always included, regardless of settings:
>
> package.json
> README
> CHANGES / CHANGELOG / HISTORY
> LICENSE / LICENCE
> NOTICE
> The file in the "main" field

I guess you really only need "yarn.lock"? Weird.

Comment (thetimothyp) [reply package.json]: hmm yeah looks like it, I was referencing the express package.json but looks like we only need index.js and yarn.lock

Comment (lennyburdette) [reply package.json]: and index.js is the "main" file right? so technically that's not needed. tho i see no harm in including it

Comment (thetimothyp) [reply package.json]: ohhhhh I see yeah that's interesting, might as well take it out if it's not necessary

**Review by mlunoe — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (eventualbuddha) [reply package.json]: Why include yarn.lock?

Comment (thetimothyp) [reply package.json]: My assumption was we'd always want to include it so installing the package would always have the right dependency versions? But it looks like they actually auto-ignore other lock files, so maybe yarn.lock should be ignored as well:
image

**Pull request merged**